app/src/main/java/net/bible/android/view/activity/navigation/History.kt

```diff
@@ -96,6 +113,6 @@ class History : ListActivityBase() {
 
     companion object {
         private const val TAG = "History"
-        private const val LIST_ITEM_TYPE = android.R.layout.simple_list_item_1
+        private const val LIST_ITEM_TYPE = android.R.layout.simple_list_item_2
```

Comment:
As you are doing customization of the view in the code, and this is rather simple layout, I would like more if you would make a custom xml for this item and do the customization there.